elixir_daisy/celery_app.py

```diff
@@ -2,10 +2,16 @@
 
 from celery import Celery
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "elixir_daisy.settings")
-
```

Comment:
I find this line very weird since it will override whatever DJANGO_SETTING we initially selected at the startup time.  It probably is ineffective since the setting object is made at the creation of the app and changing this environment variable would not change anything in the runtime; otherwise, it should have broken the app imo.

Reply:
see https://docs.python.org/3/library/stdtypes.html#dict.setdefault,  it doesn't seem it would overwrite an existing value for that key.

Reply:
True, ok then I would revert this change.